lib/components/forms/NewTeamForm.js

```diff
@@ -50,13 +50,22 @@ class NewTeamForm extends React.Component {
           description: values.teamDescription
         }
         try {
-          const team = await apiRequest(null, 'put', `/teams/${canonicalTeamName}`, Team(canonicalTeamName, spec))
-          await apiRequest(null, 'put', `/teams/${canonicalTeamName}/members/${this.props.user.id}`)
-          await this.props.handleTeamCreated(team)
-          const state = copy(this.state)
-          state.submitting = false
-          this.setState(state)
-          message.success('Team created')
+          const checkTeam = await apiRequest(null, 'get', `/teams/${canonicalTeamName}`)
```

Comment:
Would `await` handle an error?

Reply:
What i mean is .. if in the rare condition you check for the team .. API errors for some reason .. but you take it as a does not exist?

Reply:
Strictly speaking, yes. Calling `await apiRequest` with method `get` will always resolve. If the API fails then the next call to create the team would (probably) also fail...